jackson/src/com/treode/jackson/message.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2014 Treode, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+package com.treode.jackson
+
+import com.fasterxml.jackson.core.JsonGenerator
+import com.fasterxml.jackson.databind.SerializerProvider
+import com.fasterxml.jackson.databind.ser.std.StdSerializer
+import com.treode.notify.Message
+
+object MessageSerializer extends StdSerializer [Message] (classOf [Message]) {
+  def serialize (value: Message, jgen: JsonGenerator, provider: SerializerProvider) {
```

Comment:
Notice that `drive.scala` provides many serializers, and so does `cohort.scala`. You can take the same shortcut here; put both Notification and Message into `notify.scala`. Many Scala projects write lots of classes per file for convenience. In large parts of Treode we don't for compilation speed. Here though, that wasn't an issue so we chose convenience. In general, look at sibling files and follow that pattern.